src/components/core/constants.js

```diff
@@ -0,0 +1,27 @@
+const ROUTES = {  
+  
```

Comment:
Consider removing the redundant empty line.

Reply:
Done